packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -310,6 +310,10 @@
             padding: 2.4rem;
             border: 1px solid var(--general-hover);
             border-radius: 0.8rem;
+
+            @include mobile {
+                border-radius: unset;
```

Comment:
In design, advertiser page stats for responsive has no border radius, fixing this in case in the future QA comes back with this issue

### Before
![Screenshot 2022-08-04 at 11 08 55 AM](https://user-images.githubusercontent.com/103016120/182755080-b66610d4-7531-48cd-95d8-04f78650cacb.png)
 
### After
![Screenshot 2022-08-04 at 11 08 55 AM](https://user-images.githubusercontent.com/103016120/182755112-9da2b11a-6367-4254-a9d7-4d2e01c74402.png)